eternalcore-plugin/run/plugins/EternalCore/lang/pl_messages.yml

```
@@ -0,0 +1,694 @@
+  # Kontenery to inaczej skrzynie, workbench itp.
+  # W tym przypadku wykorzystujemy to do informowania graczy o otwarciu kontenera np. przy komendzie /anvil
+  # {PLAYER} - Gracz który otworzył kontener
+  genericContainerOpened: []
+  genericContainerOpenedBy: "<green>► <white>Otwarto kontener przez gracza <green>{PLAYER}<white>!"
+  genericContainerOpenedFor: "<green>► <white>Otwarto kontener dla gracza <green>{PLAYER}<white>!"
+
+# Informacja zwrotna, gdy gracz zmienia język pluginu na polski
+language:
+  languageChanged: "<green>► <white>Zmieniono język na <green>Polski<white>!"
+  decorationItems:
+    - :
```

YAMLlint (1.37.1)

[warning] cannot infer indentation: unexpected token

(indentation)